### ISSUE TITLE
[Bug report] Model deems nose and mouth present when they are not really present

### ISSUE THREAD

**Issue by Cascade4183** [googlesamples/mlkit#728]
**Describe the bug**
Nose and mouth feature detected when they are not really present

**To Reproduce**
Submit the following image and check the output

<img width="980" alt="Screenshot 2023-09-11 at 1 01 13 PM" src="https://github.com/googlesamples/mlkit/assets/144387157/72e74c04-78dd-4ebb-9ed6-f021e108c8b2">
<img width="906" alt="Screenshot 2023-09-11 at 12 42 18 PM" src="https://github.com/googlesamples/mlkit/assets/144387157/a3e2428d-fae4-42a2-9d75-757f8b91b695">

**Expected behavior**

1. Nose is not tagged as present when not present
2. Mouth is not tagged as present when not present

**SDK Info:**
 - SDK Name & Version 
    Google ML Model for face detection
    implementation ("com.google.mlkit:face-detection:16.1.5")

**Smartphone:**
 - Device/Simulator: [e.g. iPhone6, Pixel 5]

**Development Environment:**
*(For Android issue feel free to skip this section)*
- IDE Eversion: [e.g. Xcode 13.1, Android Studio 4.2]
- Laptop/Desktop: [e.g. iMac Pro (2017), HP Z840 Workstation]
- Laptop/Desktop OS/version: [e.g. macOS 12.0.1]

**Comment by miworking3** [comment]
Yes, this behavior is anticipated: ML Kit aims to provide projected facial landmarks, even when they are not visibly present in the image. Users can filter these out by comparing the coordinates of the projected landmarks with the image area to ascertain their visibility within the image.
